packages/cli/lib/services/compile.service.ts

```diff
@@ -170,26 +170,33 @@ function compileImportedFile({
 
     for (const importedFile of importedFiles) {
         const importedFilePath = path.resolve(path.dirname(filePath), importedFile);
-        const importedFilePathWithoutExtension = path.join(path.dirname(importedFilePath), path.basename(importedFilePath, path.extname(importedFilePath)));
```

Comment:
I don't follow where the bug is. `path.basename('a/b/c/schema.zod.ts', path.extname('ts'))) `should give `a/b/c/schema.zod` no?

Reply:
It's because extension-less imports are valid in Typescript, so when you autocomplete, it often imports this `schema.zod` and since we are less smart, we think .zod is the extension.

Reply:
got it